tensorboard/webapp/hparams/_redux/hparams_reducers_test.ts

```diff
@@ -309,4 +310,424 @@ describe('hparams/_redux/hparams_reducers_test', () => {
       });
     });
   });
+
+  describe('dashboardHparamColumnAdded', () => {
+    const fakeColumns = [
```

Comment:
Could you please add a couple more columns here?

Reply:
Sure thing, there are now 4.